modules/ROOT/pages/variable-transformer-reference.adoc

```diff
@@ -76,9 +76,13 @@ These examples set the variable to a Boolean value, `true`:
 * Value =
 ** `true` in Design Center, `#[true]` in Anypoint Studio
 ** `true as Boolean` in Design Center, `#[true as Boolean]` in Anypoint Studio
-** `(1 + 1 == 2)` evaluates to `true` in Design Center, `#[(1 + 1 == 2)]` evaluates to `true` in Anypoint Studio
+** `(1 + 1 == 2)` evaluates to `true` in Design Center, `#[(1 + 1 == 2)]` evaluates to `true` in Anypoint Studio.
 
-To display the value of a variable through the Logger component in Design Center, you might need to use the Anypoint Studio syntax to display the value of a variable, for example, `#[vars.myVar]` instead of `vars.myVar`.
+This example sets a variable to a Java object by using a DataWeave script. The script outputs the object in Java format:  `<set-variable value='#[%dw 2.0&#10;output application/java&#10;---&#10;{&#10;    name: "Tomo",&#10;    lastName: "Chibana",&#10;    expirationDate: now(),&#10;    salesRepr: {&#10;        name: "Mariano",&#10;        lastName: "de Achaval",&#10;    }&#10;} as Object {class: "Customer"}]' doc:name="Set Variable" variableName="Variable to Java Object"/>`.
```

Comment:
There are some double and even triple blank spaces in the code. I guess it is intentional, right?

Reply:
I copied the code as it appeared in AnypointStudio. As it is an official tool, I left it as it was.
Considering that it might be easier on the eye and that it does not affect modifying the spacing after the ";", I will take your advice and edit it only leaving 2 spaces after each ";". Thank you!!